packages/library/build.gradle.kts

```diff
@@ -189,3 +190,28 @@ realmPublish {
     }
     ojo { }
 }
+
+tasks.dokkaHtml.configure {
```

Comment:
I don't think this is enough to make the docs part of the maven publications, we had to do extra configs in Realm Java at least, but I can see that is still missing from your todo. You might want to see 

https://github.com/realm/realm-java/blob/master/realm/realm-library/build.gradle#L424 and https://github.com/realm/realm-java/blob/master/realm/realm-library/build.gradle#L252

Reply:
I added the `library-<version>-dokka.jar` to the common publication, but I do not really know if it makes sense as I don't think the IDE can consume it. I also tried with `javadoc` classifier without luck. 

The source jars are automatically published for the various platform variants and I manually added it for the common parts.

At least there is now some browsable option in the IDE and a downloadable bundle until we come up with something better.